userspace/libsinsp/test/container_engine/container_parser_cri_containerd.ut.cpp

```diff
@@ -730,11 +731,14 @@ TEST_F(sinsp_with_test_input, container_parser_cri_containerd)
 
 	ASSERT_EQ(get_field_as_string(evt, "k8s.ns.name"), "default");
 	ASSERT_EQ(get_field_as_string(evt, "k8s.pod.name"), "nginx-sandbox");
-	// todo @incertum fix pod id with next refactor, pod.id is sandbox id not pod uid
-	ASSERT_EQ(get_field_as_string(evt, "k8s.pod.id"), "hdishddjaidwnduw9a43535366368");
+	ASSERT_EQ(get_field_as_string(evt, "k8s.pod.id"), "hdishddjaidwnduw9a43535366368"); // legacy pod UID
+	ASSERT_EQ(get_field_as_string(evt, "k8s.pod.uid"), "hdishddjaidwnduw9a43535366368");
```

Comment:
Just to be more future proof
```suggestion
	ASSERT_EQ(get_field_as_string(evt, "k8s.pod.uid"), get_field_as_string(evt, "k8s.pod.id"));
```